src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlServerKeyVaultKey.md

```diff
@@ -40,6 +40,19 @@ CreationDate      : 1/1/2017 12:00:00 AM
 
 ## PARAMETERS
 
+### -AsJob
+Run cmdlet in the background```yaml
```

Comment:
For all help files, please move the ```yaml tag down to the next line (as seen in all other parameter blocks).  This is a platypus bug.

Reply:
This is a very big hassle.
Is there a platyPS issue tracking this? any idea when its going to be fixed?

Reply:
I can't seem to find the issue, though I could've sworn that I made sure it existed when I was dealing with it last month. [Here is the new issue I filed](https://github.com/PowerShell/platyPS/issues/333),